app/templates/introduction.html

```diff
@@ -44,7 +44,8 @@ <h2 class="saturn col-10@l">{{legal_text}}</h2>
               <dd class="dl__data mars">{{ meta.respondent.address.trading_as }}</dd>
             {% endif %}
             <dt id="details-changed-title" class="dl__title mercury">Change in details</dt>
-            <dd class="dl__data mars">Call <a href="tel:0300 1234 931" aria-describedby="details-changed-title">0300 1234 931</a> or email <a href="mailto:surveys@ons.gov.uk?subject=Change%20of%20details%20reference%20{{ meta.respondent.respondent_id }}" aria-describedby="details-changed-title">surveys@ons.gov.uk</a> if there have been any changes to your business name, address or structure</dd>
+               {% set ariadescibedby="details-changed-title" %}
```

Comment:
If we were going to do it this way, this variable would need to be named more appropriately e.g. `telephone_contact_aria_describedby`.

But I think we should look at creating a macro to make this clearer:
```
{{helpers.telephone_number('details-changed-title'}}
```

In helpers.html:
```
{%- macro telephone_number(aria_describedby=None, class=None) -%}
  {% set attr = {
    "aria-describedby": aria_describedby,
    "class": class
  } %}
<a href="tel:03001234931" {{attr|xmlattr}}">0300 1234 931</a>
{%- endmacro -%}
```

Reply:
Incorporated and tested. Neater solution